public/globalSolutions/dfda/right-to-trial-act.md

```
@@ -64,14 +65,15 @@ The platform enables:
 
 ### 2.3 🤖 FDAi: Autonomous Agent
+
 The FDAi continuously:
 
 * 🧠 **Analyzes** all available research and patient data
 * 📊 **Quantifies** positive and negative effects of:
-  * 💊 Pharmaceutical drugs
-  * 🍎 Food products and ingredients
-  * 🥗 Dietary patterns
-  * 🧪 Treatment combinations
+    * 💊 Pharmaceutical drugs
```

Markdownlint (0.35.0)

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)